papers.md

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
## Android UI automation test
Automatic Android GUI Traversal with High Coverage     IEEE（2014）
Targeted and depth-first exploration for systematic testing of android apps     OOPSLA（2013）
Automated testing with targeted event sequence generation     ISSTA（2013）
Automatic and scalable fault detection for mobile applications     Mobisys（2014）
Mobiguitar - a Tool for Automated Model-based Testing of Mobile Apps     2014
DynaDroid : dynamic binary instrumentation based app behavior monitor     SIN （2015）
Mining Android App Usages for Generating Actionable GUI-Based Execution Scenarios     MSR（2015）
Sapienz : multi-objective automated testing for Android applications      ISSTA （2016）
Automated Test Input Generation for Android : Are We There Yet？     ASE（2015） 
PUMA: Programmable UI-Automation for Large Scale Dynamic Analysis of Mobile Apps
AppsPlayground: Automatic Security Analysis of Smartphone Applications     CODASPY（2013）
Are these Ads Safe: Detecting Hidden Attacks through the Mobile App-Web Interfaces    NDSS （2016）
SmartDroid: an Automatic System for Revealing UI-based Trigger Conditions in Android Applications     SPSM （2012）

## Android SSL Security
Why Eve and Mallory Love Android: An Analysis of Android SSL (In)Security     CCS （2012）
Rethinking SSL Development in an Appified World      CCS （2013）
SMV-HUNTER: Large Scale, Automated Detection of SSL/TLS Man-in-the-Middle Vulnerabilities in Android Apps     NDSS （2014）
To Pin or Not to Pin—Helping App Developers Bullet Proof Their TLS Connections     USENIX （2015）
CertShim: Securing SSL Certificate Verification through Dynamic Linking     2014
SSLINT: A Tool for Detecting TLS Certificate Validation Vulnerabilities      2016
Automatically Detecting SSL Error-Handling Vulnerabilities in Hybrid Mobile Web Apps     ASIA CCS （2015）
Danger is My Middle Name: Experimenting with SSL Vulnerabilities in Android Apps     WISEC （2015）
Using Frankencerts for Automated Adversarial Testing of Certificate Validation in SSL/TLS Implementations     IEEE （2014）

##Android Ads(third) Library Security